tools/mpremote/mpremote/pyboardextended.py

```diff
@@ -459,8 +461,9 @@ def do_read(self):
     def do_seek(self):
         fd = self.rd_s8()
         n = self.rd_s32()
+        whence = self.rd_s8()
         # self.log_cmd(f"seek {fd} {n}")
-        self.data_files[fd][0].seek(n)
+        self.data_files[fd][0].seek(n, whence)
```

Comment:
I found out that seek doesn't like non-zero n for whence != 0, and in such a case (eg `f.seek(10, 1)`) this breaks down.

Reply:
Can you elaborate what you found? N here is the offset by which to seek, right? I have not have trouble myself? N = 0 would be a no-op?

Reply:
Running Python locally on my PC I get:
```
$ python
Python 3.9.5 (default, May 24 2021, 12:50:35) 
[GCC 11.1.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> f=open("README.md")
>>> f.seek(10, 1)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
io.UnsupportedOperation: can't do nonzero cur-relative seeks
>>> 
```
This `io.UnsupportedOperation` filters through to mpremote and crashes it.  This error must be a very new thing in Python...

Reply:
I see, I found something. It seems in text mode only SEEK_SET is supported on Python. I don't often use text mode, so I missed this.

I will have to see how this case can be caught.